docs/legacy/rke1_windows.md

```diff
@@ -0,0 +1,112 @@
+2. RKE1 has an **imperative** model of installation / upgrade relying on an **external** host (i.e. your computer or the Rancher server) running commands like `rke up` to get the nodes up and running (which also requires the external host to be capable of SSHing into each of the nodes), whereas RKE2 / k3s follow a **declarative** model of installation / upgrades that happen directly on the host (via registering themselves as a systemd unit that watches for changes to a configuration file). This makes RKE2 more suitable for solutions like booting up nodes via cloud-init or Terraform.
+3. RKE1 supports different CNI plugin integrations (Canal, Flannel, Calico, and Weave; only Flannel was supported for Windows), whereas RKE2 supports a different set (Cilium, Calico, Canal and Multus; only Canal is supported for Windows).
+4. RKE1 will make Docker client calls to deploy system components (kubelet, kube-proxy, etc.) as **Docker containers** (which means they can be debugged with commands like `docker ps` or `docker log`), whereas RKE2 registers itself as a systemd unit and deploys system components as **static pods managed by kubelet itself** (which means it's debugged using `crictl ps`, since containerd is the underlying container runtime for RKE2 / k3s).
+
+There are other underlying differences if you look deeper into the design (i.e. how RKE2 uses port 9345 as a sort of "service mesh" port on controlplane/etcd nodes to allow Kubernetes components on other nodes to access components like the API Server and etcd for CNI upgrades), but those won't be discussed in this document.
+
+### How does RKE1 Windows work?
+
+To support deploying RKE1 on Windows hosts, there were three primary issues that needed to be solved:
+1. Since Windows has historically had the requirement that Windows container images need to be built on the exact version of the OS it will be deployed on (i.e. Windows 2019 container images cannot be deployed on Windows 20H2 hosts), you needed to build Windows images per supported Windows Server version to deploy the Kubernetes components.
+2. Docker Engine for Windows does not support **privileged** containers (i.e. containers that run as host processes with host-level privileges in the host network)
+3. Docker Enginer for Windows does not support **hostNetwork** containers (i.e. containers that do not need host-level privileges but need to be on the host network to make requests to internal services, such as the [Windows Management Instrumentation](https://learn.microsoft.com/en-us/windows/win32/wmisdk/wmi-start-page) service to collect node-level statistics for monitoring)
+
```

Comment:
The final and most important issue here is about DockerShim. All currently supported releases of Kubernetes have dropped DockerShim from the kubelet, and now require cri-dockerd, which is supported by Mirantis. Microsoft does not recommend using cri-dockerd, in favor of using containerd instead. See https://techcommunity.microsoft.com/t5/apps-on-azure-blog/dockershim-deprecation-and-aks/ba-p/3055902

Since Microsoft does not want to support cri-dockerd on windows, we don't either.

Reply:
thank you! added